oauth2client/contrib/django_util/models.py

```diff
@@ -16,6 +16,12 @@
 
 import base64
 import pickle
+import sys
+
+try:
+    import jsonpickle
+except ImportError:
+    pass
```

Comment:
It's used in `contrib.django_util.views.py`. It's not installed with oauth2client, but necessary to use the django contrib.

What we really want is something alone the lines of making the Django contrib package an optional dependency, which brings in jsonpickle with it. For the purpose of this PR I would just assume it's installed.